blocks/blockstore/bloom_cache_test.go

```diff
@@ -104,11 +104,11 @@ func TestHasIsBloomCached(t *testing.T) {
 	block := blocks.NewBlock([]byte("newBlock"))
 
 	cachedbs.PutMany([]blocks.Block{block})
-	if cacheFails != 2 {
-		t.Fatalf("expected two datastore hits: %d", cacheFails)
+	if cacheFails != 1 {
```

Comment:
It was to because `PutMany` on default blockstore was accessing datastore two times, once checking `Has` for a key and then calling `Put`.

This means that now we don't check the datastore with `Has` we just `Put` the value.